packages/terra-theme-provider/package.json

```diff
@@ -28,7 +28,8 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
-    "terra-mixins": "^1.0.0"
+    "terra-mixins": "^1.0.0",
+    "terra-theme-properties": "file:../../terra-theme-properties-0.1.0.tgz"
```

Comment:
This will be updated to use the released version of `terra-theme-properties` once it is available.